app/src/main/java/org/sopt/and/ui/signup/SignUpScreen.kt

```diff
@@ -0,0 +1,194 @@
+import org.sopt.and.ui.component.textField.WaaveTextField
+import org.sopt.and.ui.signup.component.InfoText
+import org.sopt.and.ui.theme.ANDANDROIDTheme
+import org.sopt.and.util.isButtonEnabled
+import org.sopt.and.util.isEmailValid
+import org.sopt.and.util.isPasswordValid
+
+@Composable
+fun SignUpScreen(
+    onSignUpClick: (String, String) -> Unit = { _, _ ->}
+) {
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+
+    Scaffold(
+        topBar = { SignUpTopBar() },
+        bottomBar = {
+            SignUpBottomBar(
+                isEnabled = isButtonEnabled(email, password),
+                onSignUpClick = { onSignUpClick(email, password) }
+            )
+        },
+        containerColor = Color.Black,
+        content = { padding ->
+            SignUpContent(
+                padding = padding,
+                email = email,
+                onEmailChanged = { email = it },
+                password = password,
+                onPasswordChanged = { password = it },
+            )
+        }
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun SignUpTopBar() {
+    CenterAlignedTopAppBar(
+        title = {
+            Text(
+                text = "회원가입",
+                color = Color.White,
+                fontSize = 18.sp
+            )
+        },
+        actions = {
+            IconButton(
+                onClick = {}
+            ) {
+                Icon(
+                    imageVector = Icons.Default.Clear,
+                    contentDescription = "",
+                    tint = Color.White
+                )
+            }
+        },
+        colors = TopAppBarDefaults.topAppBarColors(Color.Black)
+    )
+}
+
+@Composable
+private fun SignUpBottomBar(
+    isEnabled: Boolean,
+    onSignUpClick: () -> Unit = {}
+) {
+    BottomAppBar(
+        containerColor = if (isEnabled) Color.Blue else Color.Gray,
+        windowInsets = BottomAppBarDefaults.windowInsets,
+        modifier = Modifier.height(60.dp)
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxSize()
+                .clickable(enabled = isEnabled) { onSignUpClick() }
+        ) {
+            Text(
+                text = "Wavve 회원가입",
+                color = Color.White
+            )
+        }
+    }
+}
+
+@Composable
+fun SignUpContent(
+    padding: PaddingValues,
+    email: String,
+    onEmailChanged: (String) -> Unit,
+    password: String,
+    onPasswordChanged: (String) -> Unit,
+) {
+    var showPassword by remember { mutableStateOf(false) }
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(padding)
+            .padding(vertical = 15.dp, horizontal = 25.dp)
+    ) {
+        Text(
+            text = "이메일과 비밀번호만으로" +
+                    "\nWavve를 즐길 수 있어요!",
```

Comment:
```suggestion
            text = "이메일과 비밀번호만으로\nWavve를 즐길 수 있어요!",
```
그냥 이렇게 적을 수도 있을 것 같네요!